.github/CODEOWNERS

```diff
@@ -8,3 +8,5 @@ components/select/* @angelia-yuqi-personal
 components/button/* @chain-txz
 components/card/* @angelia-yuqi-personal
 components/tooltip/* @yqrashawn
+components/table/* @yqrashawn
+pages/components/*/table.mdx @yqrashawn
```

Comment:
you can add others component mdx codeowner together

Reply:
Already added them in the popover pr